odoo/tools/config.py

```diff
@@ -314,6 +314,11 @@ def __init__(self, fname=None):
         group.add_option("--max-cron-threads", dest="max_cron_threads", my_default=2,
                          help="Maximum number of threads processing concurrently cron jobs (default 2).",
                          type="int")
+        group.add_option("--limit-time-cron-conn", dest="limit_time_cron_conn", my_default=0,
```

Comment:
I genuinely think that we should put some default value, here 7 days, but maybe we should do that only on master so the server behavior doesn't change in stable?
```suggestion
        group.add_option("--limit-time-cron-conn", dest="limit_time_cron_conn", my_default=60 * 60 * 24 * 7,
```

Reply:
7 days is likely too much already for high traffic databases, I would also not change it in stable but I would honestly suggest 6 hours by default.
Worst case the cron will take ~1 more minute to be processed and the cost of restarting the worker shouldn't be too high either.